xml-web-services/2.3/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Jakarta XML Web Services Specification 2.3"
+date: 2019-07-22T17:31:37+0200
+summary: "First release for Jakarta EE 8"
+---
+Jakarta XML Web Services defines a means for implementing XML-Based Web Services
+based on Jakarta SOAP with Attachments and Jakarta Web Services Metadata.
+
+* [Jakarta XML Web Services 2.3 Specification Document](./jakarta.xml.ws-spec-2.3.pdf) (PDF)
+* [Jakarta XML Web Services 2.3 Specification Document](./jakarta.xml.ws-spec-2.3.html) (HTML)
+* [Jakarta XML Web Services 2.3 Javadoc](./apidocs)
+* [Jakarta XML Web Services 2.3 TCK](http://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-xml-ws-tck-2.3.0.zip)
```

Comment:
there is no TCK starting with 'jakarta-' prefix in https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/ folder. Do you insist on changing this just for all remaining APIs (JAF/SAAJ/JAXB/JAXWS)?

Reply:
Going forward, all of the TCKs will be renamed to be jakarta-*.

Reply:
As a sort term hack, the publish job does the renaming.
We've been fixing up the web page links after the fact, but
it would be better if we can fix them as part of the submission,
although if that's the only issue here I would not hold up the
approval for it.

Reply:
I agree.  We won't hold up the approval due to this requested name change since we have the processing hack in place.  It would be good to set the proper precedent though.

Reply:
@kwsutter TCKs will be moved to proper locations once https://bugs.eclipse.org/bugs/show_bug.cgi?id=559578 gets resolved

Reply:
done